facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -51,7 +57,10 @@ object PressedCollectionDeduplication {
       collectionV: PressedCollectionVisibility,
   ): PressedCollectionVisibility = {
     // Essentially deduplicate the backfill of collectionV using header values values from accum's elements curated and backfill
-    val accumulatedHeaderURLsForDeduplication: Seq[String] = getHeaderURLsFromCuratedAndBackfilledAtDepth(accum, 10)
+    val accumulatedHeaderURLsForDeduplication: Seq[String] =
+      getHeaderURLsFromCuratedAndBackfilledAtDepth(accum, 10) ++ getHeaderURLsFromTrailsList(
+        collectionV.pressedCollection.curated,
```

Comment:
I think this is my inexperience with this code, but does this function (`deduplicateCollectionAgainstAccumulator`) de-duplicate only the backfill, or the backfill & the curated content?

Reply:
the existing comments and test cases seem to suggest that cards will only ever be removed from backfill, not curated. I'm pretty sure that when I was working on this last week I saw that that is how it works, but can't remember the details. I'll look again when I revisit it 👍